users-api/app.js

```diff
@@ -1,5 +1,16 @@
+
+//app.js
+//Bootstrapping the express application
+//load all routes
+
 // app.js
 // Bootstrapping the express application
+<<<<<<< HEAD
```

Comment:
Fix this conflict https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/